src/platform/user/authentication/utilities.js

```diff
@@ -50,7 +50,7 @@ export function sessionTypeUrl({
   queryParams = {},
 }) {
   // force v1 regardless of version
```

Comment:
Can probably remove this comment

Reply:
Good call I'll update it on Monday. Have a good weekend 🎃

Reply:
Thanks, you too!